frontend/src/components/common/theme.ts

```diff
@@ -50,3 +50,11 @@ export enum BLUE {
   E = "#73B7FF",
   F = "#DFEFFF",
 }
+
+export enum RED {
```

Comment:
We can't use Blueprint's `Colors` object, since Blueprint only has 5 colors and our `BLUE` has 6 colors!